catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -159,15 +174,20 @@ public QueryRequest createQueryRequest(String localSource, FilterBuilder filterB
     Query query =
         new QueryImpl(createFilter(filterBuilder), start, count, sortBys.get(0), true, timeout);
 
-    String source = parseSrc(localSource);
-
     QueryRequest queryRequest;
-    if (CACHE_SOURCE.equals(source)) {
-      queryRequest = new QueryRequestImpl(query, true);
-      queryRequest.getProperties().put("mode", CACHE_SOURCE);
+    if (srcs != null && srcs.length != 0) {
+      parseSrcs(localSource);
+      queryRequest = new QueryRequestImpl(query, Arrays.asList(srcs));
```

Comment:
✏️ `Collections.singletonList`

Reply:
In this `if` block `srcs` will likely have more than one string contained in it.